static/src/js/builder.jsx

```diff
@@ -258,11 +258,12 @@ function Row(props) {
     toggleVisibility,
   } = props;
 
-  const style = { display: isVisible ? "flex" : "none" };
-  const className = row.pipes.length === 0 ? "mt-2" : "mt-0";
+  const visibility = isVisible ? "d-flex" : "d-none";
```

Comment:
TIL!

Reply:
Relatedly (but not useful for this situation I think) there's the [classnames package](https://www.npmjs.com/package/classnames) which can be useful for building up `className` values dynamically.